Sources/MapboxNavigation/NavigationMapView.swift

```diff
@@ -5,6 +5,12 @@ import MapboxDirections
 import MapboxCoreNavigation
 import Turf
 
+
+private enum RouteDurationAnnotationTailPosition: Int {
```

Comment:
This probably doesn’t need a raw type. It can also be a closed enumeration.

Reply:
We do need the raw type (I believe) so that the raw value can be set in the feature properties dictionary in order to be picked up by the map runtime styling.